buildSrc/src/main/java/Dependencies.kt

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // Synchronized version numbers for dependencies used by (some) modules
-private object Versions {
+object Versions {
```

Comment:
I didn't follow why this needs the visibility change. 🤔

Reply:
Ah, because of an earlier iteration. Not needed anymore. :)